rust-runtime/aws-smithy-http-server/src/routing/lambda_handler.rs

```diff
@@ -59,24 +63,28 @@ fn convert_event(request: Request) -> HyperRequest {
     let mut path = String::from(parts.uri.path());
 
     if !raw_path.is_empty() && raw_path != path {
+        tracing::trace!(raw_path, path, "rewriting request URI to use raw HTTP path");
```

Comment:
Do we want to take this out of the `if` block?

Reply:
Why? In this branch we're rewriting the URI, but in the `else` branch we're not. In both cases we log the values, but the message is different.

Reply:
I didn't see the `tracing::trace!(raw_path, path, "not rewriting request URI");` in `else`. Ignore me.